tree/ntuple/v7/inc/ROOT/RField.hxx

```diff
@@ -2388,6 +2389,47 @@ public:
    void AcceptVisitor(Detail::RFieldVisitor &visitor) const final;
 };
 
+/// TObject requires special handling of the fBits and fUniqueID members
+template <>
+class RField<TObject> final : public RFieldBase {
+   class RTObjectDeleter : public RDeleter {
+   public:
+      void operator()(void *objPtr, bool dtorOnly) final;
+   };
+
+   static TClass *GetClass();
```

Comment:
I wonder whether the `GetClass` method can be removed since it's an alias to `TObject::Class` in any case.